tapdb/assets_store.go

```diff
@@ -202,6 +205,14 @@ type ActiveAssetsStore interface {
 	// serialized outpoint.
 	DeleteManagedUTXO(ctx context.Context, outpoint []byte) error
 
+	// LeaseManagedUTXO leases a managed UTXO identified by the passed
+	// serialized outpoint.
+	LeaseManagedUTXO(ctx context.Context, arg LeaseManagedUTXO) error
+
+	// ReleaseManagedUTXO releases a managed UTXO identified by the passed
```

Comment:
A biit of am ambiguity here re: release vs re-lease. Maybe we should move to unlock/lock?

Reply:
I called it `DeleteUTXOLease` here but will try to make it clear that this unlocks a UTXO on the higher levels. Wanted to re-use as much of the terminology we use in `lnd` (even if it's not optimal) to avoid confusion.